### DISCUSSION TITLE
Национальность (extension vs. Observation)

### DISCUSSION THREAD

**Discussion by shamilnn** [fhir-ru/core#52] [Административная РГ]
Некоторые болезни генетически обусловлены национальностью пациента, вне зависимости от того, где живет больной к моменту начала заболевания.

Нужен ли признак "национальность"? 

Если да, то является ли это частью демографических данных пациента, и тогда это extension (0..1, Must Support) в _Patient_, или может быть задан как _Observation_ с _.code_ = 46463-6 | Race or Ethnicity | LOINC ?

**Comment by vadi2** [comment]
Рекомендую как расширение для консистенции, немцы так сделали - https://simplifier.net/guide/GermanCoronaConsensusDataSet-ImplementationGuide/Ethnicgroup-duplicate-2

**Comment by niquola** [comment]
У нас в России вроде не трэкают национальность

**Reply by vadi2** [reply]
Ещё - ведь гетенически смысл имеет.

**Reply by toropok** [reply]
Как это? Конечно - да. Часто это синонимично гражданству

**Comment by toropok** [comment]
Это прикол? Вы откуда эту ересь берёте про геном и национальность?

Вы бы может по фану хоть определения слов посмотрели?

Для этих задач заложена раса и этнос (этническая группа)

**Comment by shamilnn** [comment]
Тогда можно взять более точные определения (по Дробышевскому):
**раса** - генетическое деление
**этнос** - культурное деление
**национальность** - самоопределение индивида
**нация** - политико-географическое деление

Применительно к FHIR вопрос остаётся тот же - делать как extension или как Observation?
